### PR TITLE
Install and setup pyot

### LINKED ISSUE #16: Tournament API
Utilize the riot games tournament api to create a lobby for a match when one is ready. This will both simplify the process for players, and allow us to automate the scoring of a match.

A tournament code should be generated when the game is found and the bot should post it to the channel (or DM it to users, though this may violate policies). The code/match id should also be saved into the database.

A background task will query active games and check the match history API for a game result. When a game is completed, wins and losses can be determined automatically. This should remove the `!won` command. The `!admin won` command can remain, however it should make sure that the background task does not continue checking the status of the affected game.

### PR DESCRIPTION
**Pull request by adamschachne**

### CODE REVIEW

File: inhouse_bot/cogs/admin_cog.py [hunk 1]
```diff
@@ -194,22 +193,21 @@ async def verify(self, ctx: commands.Context, user: discord.Member, *name_arg: s
...
+            summoner = await get_summoner_by_name(name)
+        except PyotException as ex:
+            if ex.code == 404:
                 return await ctx.send(f"Couldn't find Summoner name: {name}")
             # anything else gets raised
             raise ex
 
-        puuid = summoner["puuid"]
-        summoner_name = summoner["name"]
+        summoner_name = summoner.name
```
Comment (classBabacar) [inline inhouse_bot/cogs/admin_cog.py]: Better just to make var ```summoner_puuid``` just to follow the flow of the other code.

File: inhouse_bot/common_utils/lol_api/pyotconf.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import os
+from pyot.conf.model import activate_model, ModelConf
+from pyot.conf.pipeline import activate_pipeline, PipelineConf
+
+
+@activate_model("lol")
```
Comment (classBabacar) [inline inhouse_bot/common_utils/lol_api/pyotconf.py:6]: can consider putting these values in docker compose

File: inhouse_bot/common_utils/lol_api/tasks.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        history = await summoner.match_history.get()
+        for match in history.matches[:10]:
+            await queue.put(match.get())
+        first_10_matches: List[lol.Match] = await queue.join()
+    wins = []
+    for match in first_10_matches:
+        for participant in match.info.participants:
+            if participant.puuid == summoner.puuid:
+                wins.append(int(participant.win))
+    return statistics.mean(wins or [0])
```
Comment (classBabacar) [inline inhouse_bot/common_utils/lol_api/tasks.py]: newline

File: inhouse_bot/cogs/admin_cog.py (hunk 1 above)
Comment (adamschachne) [reply inhouse_bot/cogs/admin_cog.py]: good catch

File: inhouse_bot/common_utils/lol_api/pyotconf.py (hunk 2 above)
Comment (adamschachne) [reply inhouse_bot/common_utils/lol_api/pyotconf.py:6]: true, but for now the only region is NA1

File: inhouse_bot/common_utils/lol_api/tasks.py (hunk 3 above)
Comment (adamschachne) [reply inhouse_bot/common_utils/lol_api/tasks.py]: fixed. We can install some linter or something to fix these problems

File: inhouse_bot/common_utils/lol_api/tasks.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+import statistics
+
+from pyot.core.queue import Queue
+from pyot.models import lol
+
+
+async def get_summoner_by_name(summoner_name: str):
+    summoner = await lol.Summoner(name=summoner_name).get()
+    return summoner 
+
```
Comment (classBabacar) [inline inhouse_bot/common_utils/lol_api/tasks.py:11]: spacing in between functions, can we have 1 space between each

Comment (adamschachne) [reply inhouse_bot/common_utils/lol_api/tasks.py:11]: I don't think it's worth nit picking these kinds of things. A style enforcer can decide how to format code. 2 spaces seems to be conventional for python as well.

Comment (classBabacar) [reply inhouse_bot/common_utils/lol_api/tasks.py:11]: ok

**Review by classBabacar — APPROVED** [review]

**Pull request merged**